lib/src/core/transform/operation.dart

```diff
@@ -235,28 +235,31 @@ class UpdateTextOperation extends Operation {
 
 // TODO(Lucas.Xu): refactor this part
 Path transformPath(Path preInsertPath, Path b, [int delta = 1]) {
-  if (preInsertPath.length > b.length) {
-    return b;
-  }
-  if (preInsertPath.isEmpty || b.isEmpty) {
+  if ((preInsertPath.length > b.length) ||
+      (preInsertPath.isEmpty || b.isEmpty)) {
```

Comment:
```suggestion
  if (preInsertPath.length > b.length || preInsertPath.isEmpty || b.isEmpty) {
```
Don't know why I consolidated it like that, the above suggestion is as good.